README.md

```diff
@@ -2,31 +2,120 @@
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Explain the differences between encoding, encrypting, and hashing
+- Introduce password hashing in PostgreSQL.
 
 ## Introduction
 
-A 1-2 sentence summary of what will be covered.
+Let's talk about passwords.
 
-## Topic 1
+Passwords are considered sensitive information as it could help a hacker login
+to an account. For example, if the password used for an online bank account got
+out, that would be bad news. When creating applications that handle passwords
+and storing these passwords, we need to be careful with how this information is
+passed around.
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+So far, we have been storing our passwords in plain text. Look at our database
+where we stored the passwords for our `mary` and `admin` users in plain, visible
+text.
```

Comment:
I think if we change the other lesson to encode the password before persisting it, this sentence might need to be updated.